sickbeard/name_parser/parser.py

```diff
@@ -658,11 +658,6 @@ def is_anime(self):
             return True
         return False
 
-    @property
-    def is_proper(self):
-        # Part of TVEpisode object and it's inserted in 'tv_episodes' table, 'is_proper' column
```

Comment:
@medariox so from where is_proper will get its value ?

as I said in IRC:
```
    /home/travis/build/pymedusa/SickRage/sickbeard/postProcessor.py line 1111 in process
      if cur_ep.proper_tags:
   AttributeError: 'TVEpisode' object has no attribute 'proper_tags'
```

TVEpisode is created using tv_episodes table
and to populate column "is_proper" that line in parser is needed